build/container/Dockerfile

```diff
@@ -75,6 +75,6 @@ LABEL com.newrelic.nri-docker.version=$nri_docker_version \
 
 RUN apk add --no-cache \
         ntpsec=1.2.0-r1 \
-        curl=7.74.0-r0
+        curl=7.74.0-r1
```

Comment:
is this intended?

Reply:
Yes, it was failing in [here](https://github.com/newrelic/infrastructure-agent/pull/343/checks?check_run_id=2130472307)

Reply:
Reverted